posts/navarroaxel/17.node-v20.md

```diff
@@ -18,9 +18,9 @@ Say goodbye to the `dotenv` package, now Node.js can load [environment variables
 node --env-file path/to/.env index.js
 ```
 
-💡 The path to the INI file is required because Node.js didn't choose a default name for the INI file.
+💡 The path to the [INI file](https://en.wikipedia.org/wiki/INI_file) is required because Node.js didn't choose a default name for the INI file.
 
-🧠 If the INI file doesn't exist the `node` process didn't fail, just start running without the environment variables.
+🧠 If the INI file doesn't exist the `node` process doesn't fail, it just starts running without the environment variables.
```

Comment:
```suggestion
🧠 If the INI file does not exist, the node process will not fail; instead, it will start running without the environment variables.
```